src/features/RodeoDashboard/RodeoForms/CreateEventModal.tsx

```diff
@@ -35,9 +35,16 @@ const CreateEventModal: React.FC<Props> = ({defaultValues, onSubmit, onClose}) =
       <DialogTitle>Create event</DialogTitle>
       <DialogContent>
         <DialogContentText>
-          Here is the create event form
+          Add events to your rodeo.
         </DialogContentText>
         <form>
+          <Container
+          sx={{
+            display: 'flex',
```

Comment:
Changing flex-direction should allow for the form elements to each be on new rows, instead of all being on one row

We could also have more than one column of form elements if that would look better